### PR TITLE
dbSUPER super enhancer adapter

### PR DESCRIPTION
**Pull request by dawit-melka**
- Implemented adapter for [dbSuper](https://asntech.org/dbsuper/download.php) super enhancers dataset

### CODE REVIEW

File: biocypher_metta/adapters/dbsuper_adapter.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import csv
+import gzip
+import pickle
+from biocypher_metta.adapters import Adapter
+from liftover import get_lifter
+
+from biocypher_metta.adapters.helpers import check_genomic_location
+# Example dbSuper tsv input files:
```
Comment (Habush) [inline biocypher_metta/adapters/dbsuper_adapter.py:8]: what does the `rank` column refer to?

File: biocypher_metta/adapters/dbsuper_adapter.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                    props = {}
+                    if self.write_properties:
+                        props['chr'] = chr
+                        props['start'] = start
+                        props['end'] = end
+                        if self.add_provenance:
+                            props['source'] = self.source
+                            props['source_url'] = self.source_url
+
+                    yield se_id, self.label, props
```
Comment (Habush) [inline biocypher_metta/adapters/dbsuper_adapter.py]: You should use `build_regulatory_region_id` function to construct the id as you did for promoters and add `se_id` as a property

File: biocypher_metta/adapters/dbsuper_adapter.py (hunk 1 above)
Comment (dawit-melka) [reply biocypher_metta/adapters/dbsuper_adapter.py:8]: The rank assigned to the super enhancer within that specific cell type based on the [H3K27ac ChIP-seq](https://en.wikipedia.org/wiki/H3K27ac) signal strength. Higher ranks indicate a stronger signal and potentially a more potent super enhancer.

**Comment by Habush** [comment]
Looks good!

**Pull request merged**